python/torch_mlir_e2e_test/torchscript/framework.py

```diff
@@ -321,7 +322,7 @@ def run_workers_in_parallel(task_queue: mp.Queue, worker):
 
     processes = []
     for i in range(NUMBER_OF_PROCESSES):
-        p = mp.Process(target=worker, args=(task_queue, ))
+        p = mp.get_context('fork').Process(target=worker, args=(task_queue, ))
```

Comment:
nit: I'm an idiot and used `'`'s instead of `"`'s in my suggestion. Personal style leaked in. Mea culpa.

Reply:
no worries. updated.